File: week1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float change;
    do
    {
        change = get_float("Change owed: ");
    }
    while (change < 0.00);
    
    
    int cents = round(change * 100);
    
    int count = 0;
    
    while (cents > 24) 
    {
        cents = cents - 25;
        count++;
    }
    while (cents > 9) 
    {
        cents = cents - 10;
        count++;
    }
    while (cents > 4) 
    {
        cents = cents - 5;
        count++;
    }
    count = cents + count;
    
    printf("Change: %d\n", count);
}

File: week6/cash/cash.py
from cs50 import get_float


def main():
    x = get_right_float()


def get_right_float():
    while True:
        change = get_float("Change owed: ")
        if change < 0:
            print("number is wrong")
        else:
            break

    cents = round(change * 100)
    count = 0

    while cents > 24:
        cents = cents - 25
        count = count + 1
    while cents > 9:
        cents = cents - 10
        count = count + 1
    while cents > 4:
        cents = cents - 5
        count = count + 1
    count = cents + count

    print("Change: ", + count)


main()
